dataland-frontend/src/components/pages/ApiKeysPage.vue

```diff
@@ -76,11 +76,7 @@
             </template>
```

Comment:
computed property "PageTitleState" should be named "pageTitleState" to be consistent with naming of computed properties in other components

Reply:
ok

Reply:
At a second glance, just "pageTitle" might be even better to read/understand